### PR TITLE
Adding top 5 shows

### PR DESCRIPTION
**Pull request by taymor10**
@mjfaade10 Here is a list of my top 5 shows and movies. Let me know what you think!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ Chappelle Show
 Breaking Bad
 How To Make It In America
 GIRLS
```
Comment (mjfaade10) [inline README.md]: @taymor10 GIRLS? There are def better shows!

Comment (taymor10) [reply README.md]: Ha! Let me think of another one...

Comment (mjfaade10) [reply README.md]: Thanks

**Review by mjfaade10 — COMMENTED** [review]
G2G

**Review by mjfaade10 — CHANGES_REQUESTED** [review]
Oh no, GIRLS is still there. Do you mind updating this with a better show :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
 # hello-world
 Testing...
+
+Top five shows:
+Succession
+Chappelle Show
+Breaking Bad
+How To Make It In America
+ATLANTA
```
Comment (mjfaade10) [inline README.md:9]: Looks good!

Comment (taymor10) [reply README.md:9]: Thanks

**Review by mjfaade10 — COMMENTED** [review]
This can merge

**Review by mjfaade10 — APPROVED** [review]
Thanks!

**Pull request merged**